ui/lesson/HashChallenge/HashPrompt.tsx

```diff
@@ -0,0 +1,42 @@
+import clsx from 'clsx'
```

Comment:
What is the purpose of this component, if its just used for markup lets move it to `Hasher` or `HashChallenge` instead.